crates/matrix-sdk/src/send_queue.rs

```diff
@@ -2138,6 +2159,8 @@ impl SendReactionHandle {
             room: self.room.clone(),
             transaction_id: self.transaction_id.clone().into(),
             media_handles: None,
+            // This isn't quite right, should this be the newly enqueued cancel event?
+            created_at: self.created_at.clone(),
```

Comment:
You're right, it should be `Some(MilliSecondsSinceUnixEpoch::now())`. I think it's a fine value to set `now()` here, since this is the time of the request to cancel.